src/main/java/com/vadmack/mongodbtest/service/ProjectService.java

```diff
@@ -22,22 +24,24 @@
 
     private final ModelMapper modelMapper = new ModelMapper();
 
-    public List<Project> findAll() {
-        return repository.findAll();
+    public List<ProjectDto> findAll() {
+        return repository.findAll().stream().map(project -> modelMapper.map(project, ProjectDto.class))
```

Comment:
`project -> modelMapper.map(project, ProjectDto.class)` is repeated several times in the sevice. I suggest to create and use  new method like this:
`ProjectDto entityToDto(Project project)`